### DISCUSSION TITLE
Cable import several terminations

### DISCUSSION THREAD

**Discussion by jorgeelima** [netbox-community/netbox#13206] [Help Wanted!]
Hello does anyone know how to create a cable using the import window that has multiple terminations? 

I am trying to use the import information but no luck

side_a_device,side_a_type,side_a_name,side_b_device,side_b_type,side_b_name,type,status
template_device_a,dcim.interface,Gi0/0/0,template_device_b,dcim.frontport,"Front Port 1,Front Port 2",mmf-om4,connected

And I get the error "Record 1 side_b_name: B side termination not found: CFTemplate-ODF-01-SH-01-FL-01 Front Port 1|Front Port 2"

What can I use in the syntax to make this work like this? 

![image](https://github.com/netbox-community/netbox/assets/79458010/f6305326-c0d5-4b6f-a5fe-0b8be2cff00a)

**Comment by jorgeelima** [comment]
Can anyone help please?

**Comment by jorgeelima** [comment]
Please can someone help on this issue?

**Comment by drobnymichal** [comment]
I am also facing this problem. I also tried importing data in JSON format. Unfortunately, it fails.

See example above. 
`And I get the error "Record 1 side_b_name: B side termination not found: CFTemplate-ODF-01-SH-01-FL-01 Front Port 1|Front Port 2"`
